package.json

```diff
@@ -10,14 +10,15 @@
     "format": "prettier --list-different --write *.{js,ts,tsx} h/static/scripts",
     "lint": "eslint h/static/scripts",
     "test": "gulp test",
+    "test:vitest": "gulp test:vitest",
```

Comment:
This is obsolete now that Vitest is the only way of running tests.

Reply:
Right! I forgot to remove this.